modules/dashboard/test/TestPlan.md

```diff
@@ -1,4 +1,22 @@
-1. Log in. Note the time. Log out and log back in after 2 minutes. Check that welcome panel info is correct.
-   [Automate Test on Travis_CI]
-2. Check performance on the raisin bread dataset: make sure the dashboard page displays within 5 seconds or less.
-   [Manual Test]
+# Dashboard Testplan
+1. Log in. Note the time. Log out and log back in after 2 minutes. Check that welcome panel info is correct. [Automate Test on Travis_CI]
+2. Check performance on the raisin bread dataset: make sure the dashboard page displays within 5 seconds or less. [Manual Test]  
+3. Verify the links on the dashboard can navigate to the corresponding pages (eg. clicking the Access Profile tab on the navigation bar will navigate to the Candidate list page). [Automated Test]  
+
+## Dashboard Widgets
+1. Verify that the dashboard (with full permissions) includes the following widgets.
+   * My tasks
+   * Recruitment
+   * Study Progression
+   * Document Repository Notifications
```

Comment:
@driusan not sure if these should also be part of individual module testing. I don't actually know if 100% of the content of the dashboard is module widgets or if some widgets belong to the dashboard itself. I figured you might know

Reply:
that's fair, i'll remove the Doc Repo bullet.   not sure if recruitment and study progression can really be tested in the same way via the (beta) candidate profiles module